monitoring/src/test/scala/akkaviz/events/EventPublisherActorTest.scala

```diff
@@ -0,0 +1,153 @@
+package akkaviz.events
+
+import akka.actor._
+import akka.dispatch.sysmsg.DeathWatchNotification
+import akka.testkit.{TestProbe, TestActorRef, ImplicitSender, TestKit}
+import akkaviz.config.Config
+import akkaviz.events.EventPublisherActor.{Unsubscribe, Subscribe}
+import akkaviz.events.types._
+import org.scalatest.{Matchers, WordSpecLike}
+
+class EventPublisherActorTest extends TestKit(ActorSystem("EventPublisherActorTestSystem")) with ImplicitSender
+    with WordSpecLike with Matchers with akkaviz.events.Helpers {
+
+  import scala.concurrent.duration._
+
+  val someActorRef = TestActorRef(new Actor { def receive = { case _ => () } }, "someActor")
+
+  def withPublisher(enabledTest: () => Boolean = () => true)(test: (ActorRef) => Any) = {
+    val publisher = system.actorOf(Props(classOf[EventPublisherActor], enabledTest, Config.maxEventsInSnapshot))
+    try { test(publisher) }
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Consider using util.Try instead of a try-catch block](https://www.codacy.com/app/lustefaniak/akka-viz/file/2355384790/issues/source?bid=3121282&fileBranchId=3217752#l20)